src/g1.rs

```diff
@@ -453,6 +454,39 @@ impl G1Affine {
     }
 }
 
+impl SerdeObject for G1Affine {
+    fn from_raw_bytes_unchecked(bytes: &[u8]) -> Self {
+        debug_assert_eq!(bytes.len(), UNCOMPRESSED_SIZE);
+        let input: [u8; UNCOMPRESSED_SIZE] = bytes.try_into().unwrap();
+        Self::from_uncompressed_unchecked(&input).unwrap()
```

Comment:
I know it is not part of this PR, but why does 'from_uncompressed_unchecked' returns an error?

Reply:
Excellent 👍